### PR TITLE
feat(runtime): write next runtime logs to stdout

### PR DESCRIPTION
**Pull request by orhun**
you know the drill 🔨🪚🛠️🔩🪛🔧🪚🔨🔩🛠️

### CODE REVIEW

File: runtime/tests/resources/axum-wasm-expanded/src/lib.rs [hunk 1]
```diff
@@ -24,12 +24,12 @@ async fn app(request: shuttle_next::Request<BoxBody>) -> shuttle_next::response:
 }
 
 async fn hello() -> &'static str {
-    debug!("in hello()");
+    println!("in goodbye()");
```
Comment (chesedo) [inline runtime/tests/resources/axum-wasm-expanded/src/lib.rs:27]:
```suggestion
    println!("in hello()");
```

**Review by chesedo — COMMENTED** [review]
Just missing the next codegen update :smile:

File: runtime/tests/resources/axum-wasm-expanded/src/lib.rs (hunk 1 above)
Comment (orhun) [reply runtime/tests/resources/axum-wasm-expanded/src/lib.rs]: d0f4a066891f876b0fd0004df81f0963e55b622b

**Comment by orhun** [comment]
Updated the codegen in 6569b3c3f0c14b1e932d18358d278dd69ab42e15

**Review by oddgrd — APPROVED** [review]
Nice, this is much simpler now. Thanks! :star_struck:

**Review by chesedo — APPROVED** [review]

**Pull request merged**